### PR TITLE
*: update basic features for 7.0

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

as title

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [X] master (the latest development version)
- [X] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/13503
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -20,212 +20,218 @@ This document lists the features supported in different TiDB versions, including
...
+| [`SET CONFIG`](/dynamic-config.md) | Y | Y | Y | Y | E | E | E | E | E | E |
+| [DM WebUI](/dm/dm-webui-guide.md) | E | E | E | E | N | N | N | N | N | N |
+| [Foreground Quota Limiter](/tikv-configuration-file.md#foreground-quota-limiter) | Y | Y | Y | E | N | N | N | N | N | N |
+| [Background Quota Limiter](/tikv-configuration-file.md#background-quota-limiter) | E | E | E | N | N | N | N | N | N | N |
+| [EBS volume snapshot backup and restore](https://docs.pingcap.com/tidb-in-kubernetes/v1.4/backup-to-aws-s3-by-snapshot) | Y | Y | Y | N | N | N | N | N | N | N |
+| [PITR](/br/backup-and-restore-overview.md) | Y | Y | Y | N | N | N | N | N | N | N |
+| [Global memory control](/configure-memory-usage.md#configure-the-memory-usage-threshold-of-a-tidb-server-instance) | Y | Y | Y | N | N | N | N | N | N | N |
+| [Cross-cluster RawKV replication](/tikv-configuration-file.md#api-version-new-in-v610) | E | E | E | N | N | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-new-in-v50) | E | E | E | E | E | E | E | E | E | N |
+| [Resource control](/tidb-resource-control.md) | Y | E | N | N | N | N | N | N | N | N |
```
Comment (ran-huang) [inline basic-features.md:233]:
```suggestion
| [Resource control](/tidb-resource-control.md) | E | E | N | N | N | N | N | N | N | N |
```

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: basic-features.md [hunk 2]
```diff
@@ -20,212 +20,218 @@ This document lists the features supported in different TiDB versions, including
...
-| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                                                   | Y | Y | Y | Y           | Y            | Y            | Y            | Y            | Y            |
-| [Stream data to Amazon S3, Azure Blob Storage, and NFS through TiCDC](/ticdc/ticdc-sink-to-cloud-storage.md) | E | E | N | N | N | N | N | N | N |
-| [TiCDC supports bidirectional replication between two TiDB clusters](/ticdc/ticdc-bidirectional-replication.md) | Y | Y | N | N | N | N | N | N | N |
+| Data import and export | 7.0 | 6.6 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Fast Importer (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| mydumper logical dumper | Deprecated | Deprecated | Deprecated | Deprecated | Deprecated | Deprecated | Deprecated | Deprecated | Deprecated | Deprecated |
+| [Dumpling logical dumper](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Transactional `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N [^5] |
```
Comment (hfxsd) [inline basic-features.md:195]:
```suggestion
| [Transactional `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | E | Y | Y | Y | Y | Y | Y | Y | Y | N [^5] |
```

**Review by Frank945946 — APPROVED** [review]

File: basic-features.md [hunk 3]
```diff
@@ -235,4 +241,6 @@ This document lists the features supported in different TiDB versions, including
 
 [^4]: Starting from [v6.4.0](/releases/release-6.4.0.md), TiDB supports [high-performance and globally monotonic `AUTO_INCREMENT` columns](/auto-increment.md#mysql-compatibility-mode)
 
-[^5]: For TiDB v4.0, the `LOAD DATA` transaction does not guarantee atomicity.
+[^5]: Starting from [TiDB v7.0.0](/releases/release-7.0.0.md), the new clauses `FORMAT`, `FIELDS DEFINED NULL BY`, `With batch_size=<number>,detached`, and support for importing data from S3 and GCS, are experimental features.
```
Comment (ran-huang) [inline basic-features.md:244]: To make it consistent with other footnotes
```suggestion
[^5]: Starting from [v7.0.0](/releases/release-7.0.0.md), the new clauses `FORMAT`, `FIELDS DEFINED NULL BY`, `With batch_size=<number>,detached`, and support for importing data from S3 and GCS, are experimental features.
```

Comment (ran-huang) [inline basic-features.md:244]: 可能只有 7.0.0 是实验特性，不需要用 starting from
```suggestion
[^5]: For [TiDB v7.0.0](/releases/release-7.0.0.md), the new clauses `FORMAT`, `FIELDS DEFINED NULL BY`, `With batch_size=<number>,detached`, and support for importing data from S3 and GCS, are experimental features.
```

**Pull request merged**